.github/workflows/darwin.yaml

```diff
@@ -16,6 +16,8 @@ name: Darwin
 
 on:
     push:
+        branches:
+            - master
```

Comment:
People absolutely trigger CI runs in their own forks (to avoid spamming the shared CI) by pushing to a non-master branch.  Does this break that?

Reply:
Also, does this disable CI for Matter "release branches" that correspond to a release, SVE, etc?